lib/system/api/containers.toit

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import uuid
+import system.services show ServiceClient
+
+interface ContainersService:
```

Comment:
I would call it ContainerService.

It's dog food, not dogs food, even though more than one dog eats it.  It's an apple tree, not an apples tree.

Reply:
Yeah, I think you're right. Will fix in follow up PR.